src/reader.py

```diff
@@ -0,0 +1,29 @@
+class Reader():
+    def __init__(self, file_name):
+        self.file_name = file_name
+        self.feature_file = []
+        self.set_feature_file()
+
+    def set_feature_file(self):
+        with open(self.file_name, "r") as f:
+            test_case = []
+            first_pass = True
+            for line in f:
```

Comment:
Could you set `line = line.strip()` here to save you doing every time?
```suggestion
            for line in f:
                line = line.strip()
```

Reply:
You can then do `if not line:` on line 12